src/engine/Settings.cs

```diff
@@ -124,7 +124,7 @@ private Settings()
     ///   Enable or disable 3D background scenes in the menu
     /// </summary>
     [JsonProperty]
-    public SettingValue<bool> Menu3DBackgroundEnabled { get; private set; } = new(true);
+    public SettingValue<bool> Menu3DBackgroundEnabled { get; set; } = new(true);
```

Comment:
This isn't correct. Changing the setting value will break all observers on the value from working. To assign a new value you need to set the underlying value `Menu3DBackgroundEnabled.Value = false` that will notify all observers that the value changed.